apps/dashboard/app/controllers/projects_controller.rb

```diff
@@ -103,12 +103,13 @@ def destroy
 
   # GET /projects/:project_id/jobs/:cluster/:jobid
   def job_details
+    project = Project.find(params[:project_id])
```

Comment:
You should avoid using `params` directly and instead us maybe `job_details_params` so the list of parameters we're accessing are known to be valid and permitted.